docs/user/batch-email-notifications.rst

```diff
@@ -0,0 +1,83 @@
+Batch Email Notifications
+=========================
+
+.. contents:: **Table of Contents**:
+    :depth: 2
+    :local:
+
+.. _notifications_batch_email_notifications:
+
+Batch Email Notifications
+-------------------------
+
+Batch email notifications help manage the volume of emails sent to users,
+particularly during periods of high alert activity. By batching emails,
+the system reduces the risk of emails being flagged as spam and prevents
+email inboxes from rejecting alerts due to overuse. The following features
+and configurations make up the batch email notification system:
+
+Batch Email Feature
+~~~~~~~~~~~~~~~~~~~
+
+The batch email notification feature ensures that:
+
+- If more than one email is sent to a specific user within a 30-minute
+  window, subsequent emails are batched into a summary.
+- The sending of individual emails is paused for 30 minutes when batching
+  is active.
+- **Note**: If new alerts arrive while a batch is pending, they are added
+  to the existing summary. However, the timer does not reset. The batch
+  email will be sent out when the initial 30-minute interval expires.
```

Comment:
in this section, I think that instead of mentioning the 30 minutes, we should specify the batch interval and link the related setting with a reference.

Reply:
Updated.